GPCS4/Graphic/Gve/GveContext.cpp

```diff
@@ -505,7 +505,8 @@ void GveContext::updateVertexBindings()
 		++bindingCount;
 	}
 
-	m_cmd->cmdBindVertexBuffers(0, bindingCount, buffers.data(), offsets.data());
+	if (bindingCount > 0)
+		m_cmd->cmdBindVertexBuffers(0, bindingCount, buffers.data(), offsets.data());
```

Comment:
single line in `if` statement should also add brace:
https://github.com/Inori/GPCS4/blame/master/Doc/CodeStyle.md#L174